async_simple/util/ThreadPool.h

```diff
@@ -128,6 +128,9 @@ inline ThreadPool::ThreadPool(size_t threadNum, bool enableWorkSteal,
     std::vector<uint32_t> cpu_ids;
     if (_enableCoreBindings)
         getCurrentCpus(cpu_ids);
+#else
+    // Avoid unused member warning.
+    (void)_enableCoreBindings;
```

Comment:
can we use `[[maybe_unused]]` attribute for member field?

Reply:
Good idea! Done.

Reply:
It would be ignored by GCC and cause another error. I would use the original style.

Reply:
ok